client/templates/userprofile/settings.js

```diff
@@ -1,151 +1,46 @@
-    },
-    'passwordsuccess': function() {
-        return Session.get("passwordsuccess");
-    },
-
-    'perrors': function() {
-        return Session.get("perrors");
-    },
-
-    'langs': function () {
-        return data = ["English", "Finnish", "German", "Persian", "Russian", "Swedish"];
-    }
-});
-
-Template.profileSettings.events({
-
-    'change #realName': function(event,template) {
-      if(template.find("#realName").value)
-      {
-          Session.set("realerror", "");
-      }
-        else
-      {
-          Session.set("realerror", "Your Realname is empty!");
-      }
-    },
-    'change #userName': function(event,template) {
-        if(template.find("#userName").value)
-        {
-            Session.set("usererror", "");
-        }
-        else
-        {
-            Session.set("usererror", "Your username is empty!");
-        }
-    },
-    'change #email': function(event,template) {
-        if(template.find("#email").value)
-        {
-            Session.set("emailerror", "");
-            if(!(isEmail(template.find("#email".value))))
-            {
-                Session.set("emailerror", "You have not entered a correct Email!");
-            }
-            else
-            {
-                Session.set("emailerror", "");
-            }
-        }
-        else
-        {
-            Session.set("emailerror", "Your Email is empty!");
-        }
-
-    },
-
-    'click #saveChanges': function(event, template) {
-        var realname = template.find("#realName").value;
-        var username = template.find("#userName").value;
-        var gender = template.find("#gender option:selected").value;
-        var language = template.find("#language option:selected").value;
-        var biography = template.find("#biography").value;
-        var email = template.find("#email").value;
-
-        if (Meteor.userId())
-        {
-            if (realname && username && gender && language && email && isEmail(email))
-            {
-                Session.set("basicsuccess", "Data successfully changed!");
-                console.log("success");
-
-                //TODO simplify
-                Meteor.call("User.update", Meteor.userId(),"realname", realname);
-                Meteor.call("User.update", Meteor.userId(),"username", username);
-                Meteor.call("User.update", Meteor.userId(),"gender", gender);
-                Meteor.call("User.update", Meteor.userId(),"email", email);
-                Meteor.call("User.update", Meteor.userId(),"language", language);
-                Meteor.call("User.update", Meteor.userId(),"biography", biography);
-
-            }
-            else
-            {
-                console.log("there has been a problem!");
-            }
-
+  profile: function() {
+    return Meteor.users.findOne({
+      'id': this._id
+    });
+  },
+  genderOptions: function() {
+    return [{
+      optgroup: "Please select your Gender",
+      options: [{
+        label: "Female",
```

Comment:
The labels for these gender options need i18n strings. The values can stay as they are, since autoForm validates against the values.

E.g. 
```
Add i18n string here. E.g.

```
mf('userProfile-genderLabels-female', 'Female');
```